src/indra_cogex/sources/indra_db/__init__.py

```diff
@@ -366,6 +375,7 @@ def get_nodes(self) -> Iterable[Node]:
                             {
                                 "evidence:string": json.dumps(evidence),
                                 "stmt_hash:long": stmt_hash,
+                                "source_api:str": evidence["source_api"],
```

Comment:
Once complication with this is that `evidence["source_api"]` will, in this setting, not always match the source counts keys due to the fact that INDRA's input sources use nomenclature not always consistent with the INDRA DB's. Given that we are using INDRA DB nomenclature elsewhere for source metadata, we should probably map to those here.

Reply:
So the motivation behind this one is to enable medscan filtering, so we could either ask @kkaris to make this addition (since he's the keeper of the ultimate source count map) or we could alternatively make it a "is_medscan:bool" flag instead